src/components/Editor/HighlightLine.js

```diff
@@ -66,7 +66,14 @@ export class HighlightLine extends Component<Props> {
       return false;
     }
 
-    return true;
+    if (
+      editorLine !== this.previousEditorLine ||
+      this.props.selectedSource !== selectedSource
```

Comment:
having the `this.props.selectedSource !== selectedSource` check in, seems to make the highlight flash a really quick, that sometimes you might not see the line highlighted.